src/main/scala/mesosphere/marathon/core/flow/LaunchTokenConfig.scala

```diff
@@ -0,0 +1,16 @@
+package mesosphere.marathon.core.flow
+
+import org.rogach.scallop.ScallopConf
+
+trait LaunchTokenConfig extends ScallopConf {
+  //scalastyle:off magic.number
+
+  lazy val launchTokenRefreshInterval = opt[Long]("launch_token_refresh_interval",
+    descr = "The interval (ms) in which to refresh the launch tokens to --launch_token_count",
+    default = Some(30000))
+
+  lazy val launchTokens = opt[Int]("launch_tokens",
+    descr = "Launch tokens per interval",
+    default = Some(1000))
```

Comment:
I guess my point here is, that this value highly depends on the cluster size.
So I am not sure, what a sensible default should look like.
I kind of like the idea, that e.g. java is using for memory: 
-either define it explicitly
-the default is 25% of available mem but not more than 1GB
I wonder, if we should do something similar..